jvm-packages/xgboost4j-spark/docs/index.md

```diff
@@ -0,0 +1,41 @@
+# XGBoost4J-Spark
+
+(3 - 4 sentences intro)
+
+# Build an Application with XGBoost4J-Spark
+
+(based on maven build), step by step (structure of program)
+
+## Data Preparation
+
+Data Frame based
```

Comment:
@yanboliang I think we can choose a classic dataset here, maybe the ones here https://github.com/dmlc/xgboost/tree/master/demo/data

Reply:
Sound good.